test_scripts/API/PerformAudioPassThru/ideas_PAPT_new_param_audioPassThruIcon.txt

```diff
@@ -0,0 +1,118 @@
+  +./test_scripts/API/PerformAudioPassThru/001_ATF_P_PerformAudioPassThru_SUCCESS_All_Params.lua 
+  Check_that_SUCCESS_returned_when_all_parameters_valid_within_bounds
+  +./test_scripts/API/PerformAudioPassThru/002_ATF_P_PerformAudioPassThru_SUCCESS_All_Params_Upper.lua
+  Check_that_SUCCESS_returned_when_all_param_valid_upper_bound 
+  +./test_scripts/API/PerformAudioPassThru/003_ATF_P_PerformAudioPassThru_SUCCESS_All_Params_Lower.lua
+  Check_that_SUCCESS_returned_when_all_param_valid_lower_bound  
+  +./test_scripts/API/PerformAudioPassThru/004_ATF_P_PerformAudioPassThru_SUCCESS_Only_Mandatory_Params_with_audioPassThruIcon
+  Check_that_SUCCESS_returned_when_only_mandatory_param_and_audioPassThruIcon_sent
+  +./test_scripts/API/PerformAudioPassThru/005_ATF_P_PerformAudioPassThru_WARNINGS_Different_Speech_Capabilities_PRE_RECORDED.lua
+  Check_that_WARNINGS_returned_when_speech_capabilities_PRE_RECORDED
+  +./test_scripts/API/PerformAudioPassThru/006_ATF_P_PerformAudioPassThru_WARNINGS_Different_Speech_Capabilities_SAPI_PHONEMES.lua 
+  Check_that_SUCESS_returned_when_speech_capabilities_SAPI_PHONEME
+  +./test_scripts/API/PerformAudioPassThru/007_ATF_P_PerformAudioPassThru_WARNINGS_Different_Speech_Capabilities_LHPLUS_PHONEMES.lua 

```

Comment:
@istoimenova 
It has been confirmed in clarification <<Expected result if TTS name in speech capabilities is not the default one "TEXT">> that this is the result code to be returned. Tests will be moved in check of result code.